[2022-06-12 16:47:29,759] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ingest_data.save_data_task scheduled__2022-06-12T01:57:00+00:00 [queued]>
[2022-06-12 16:47:29,791] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ingest_data.save_data_task scheduled__2022-06-12T01:57:00+00:00 [queued]>
[2022-06-12 16:47:29,792] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-12 16:47:29,793] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-12 16:47:29,794] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-12 16:47:29,867] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): save_data_task> on 2022-06-12 01:57:00+00:00
[2022-06-12 16:47:29,873] {standard_task_runner.py:52} INFO - Started process 5506 to run task
[2022-06-12 16:47:29,880] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'ingest_data', 'save_data_task', 'scheduled__2022-06-12T01:57:00+00:00', '--job-id', '630', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_dag.py', '--cfg-path', '/tmp/tmp_bq9li6h', '--error-file', '/tmp/tmplwsxb51t']
[2022-06-12 16:47:29,883] {standard_task_runner.py:80} INFO - Job 630: Subtask save_data_task
[2022-06-12 16:47:30,033] {task_command.py:370} INFO - Running <TaskInstance: ingest_data.save_data_task scheduled__2022-06-12T01:57:00+00:00 [running]> on host DESKTOP-GDPK8BL.localdomain
[2022-06-12 16:47:30,587] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=save_data_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-12T01:57:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-12T01:57:00+00:00
[2022-06-12 16:47:30,606] {python.py:173} INFO - Done. Returned value was: None
[2022-06-12 16:47:30,637] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=ingest_data, task_id=save_data_task, execution_date=20220612T015700, start_date=20220612T144729, end_date=20220612T144730
[2022-06-12 16:47:30,740] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-12 16:47:30,972] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
